README.md

```diff
@@ -1,36 +1,10 @@
 # Fuel Indexer
 
+![Fuel Logo](https://miro.medium.com/fit/c/176/176/1*W-1e0zVhcLui4u-ci2e-Fg.png "title")
```

Comment:
Hmmm. Unlike github and badges, Medium wasn't design to host images for other sites. Instead, include this image in a directory in this repo.